php-checks/src/main/java/org/sonar/php/checks/FormattingStandardCheck.java

```diff
@@ -172,6 +189,7 @@
   @RuleProperty(
     key = "extends_implements_line",
     defaultValue = "true",
+    description = "Enables the validation of interface indentation in call declarations.",
```

Comment:
`Enables the verification that "extends" and "implements" are on the same line as class declarations`